src/botPage/view/blockly/blocks/tools/key_value_pair.js

```diff
@@ -0,0 +1,35 @@
+import { translate } from '../../../../../common/i18n';
+import { expectValue } from '../shared';
+
+Blockly.Blocks.key_value_pair = {
+    init() {
+        this.jsonInit({
+            message0: translate('Key: %1 Value: %2'),
+            args0   : [
+                {
+                    type: 'field_input',
+                    name: 'KEY',
+                    text: 'default',
+                },
+                {
+                    type: 'input_value',
+                    name: 'VALUE',
+                },
+            ],
+            colour : '#dedede',
+            output : null,
+            tooltip: translate('Payload for webhook'),
+        });
+    },
+};
+
+Blockly.JavaScript.key_value_pair = block => {
+    const key = block.getFieldValue('KEY') || '';
+    const value = expectValue(block, 'VALUE');
```

Comment:
`value` could be empty, it's perfectly fine to have a `null` value in JSON.

Reply:
in that case, i need to remove shadow from `key_pair_value`?